exdb/models.py

```diff
@@ -66,8 +66,7 @@ class Experience(models.Model):
     created_datetime = models.DateTimeField(default=now, blank=True)
     recognition = models.ManyToManyField(Organization, blank=True)
     status = models.CharField(max_length=2, choices=STATUS_TYPES, default=STATUS_TYPES[1][0])
-    approved_timestamp = models.DateTimeField(blank=True, null=True)
-    approver = models.ForeignKey(settings.AUTH_USER_MODEL, blank=True, null=True, related_name="approver")
+    current_approver = models.ForeignKey(settings.AUTH_USER_MODEL, blank=True, null=True, related_name='approver')
```

Comment:
I feel like `approval_queue` would be a clearer related name for this than `approver`.